docs/data/material/getting-started/templates/checkout/PaymentForm.js

```diff
@@ -19,7 +19,36 @@ import CreditCardRoundedIcon from '@mui/icons-material/CreditCardRounded';
 import SimCardRoundedIcon from '@mui/icons-material/SimCardRounded';
 import WarningRoundedIcon from '@mui/icons-material/WarningRounded';
 
-import { styled } from '@mui/system';
+import { styled } from '@mui/material/styles';
+
+const StyledCard = styled(Card)(({ theme }) => ({
```

Comment:
```suggestion
const Card = styled(MuiCard)(({ theme }) => ({
```

I'd do something like this instead so we avoid using the `Styled` prefix in all of these components, which is a convention we try to follow. You'll need to change the import so that works:

```
import Card as MuiCard from '@mui/material/Card';
```